app/renderer/components/navigation/buttons/windowCaptionButtons.js

```diff
@@ -60,7 +61,7 @@ class WindowCaptionButtons extends ImmutableComponent {
     const props = { tabIndex: -1 }
 
     return <div className={cx({
-      fullscreen: this.props.windowMaximized,
+      fullscreen: this.props.windowMaximizedFullScreen,
```

Comment:
maximized and fullscreen are only the same thing on macos. We should specify one or the other

Reply:
I preserved the same functionality, I didn't change anything. So it's working the same way it was before. The only thing that I changed is naming, because we had/have both things in it

Reply:
I'm only referring to the naming. We have maximized and fullscreen in one property? That's a bit confusing imo

Reply:
Yeah this is how we have it now :smiley: 

https://github.com/brave/browser-laptop/blob/6fe9849cd3aea3faddf379b96fddb7abdecdd247/app/renderer/components/main/main.js#L533

Reply:
what do you propose? split it and use it separately?

Reply:
the existing logic seems wrong to me because fullscreen and maximized should not be treated the same way

Reply:
Current logic is working (no reports about any problems), so I would suggest that if it's not working we should open new issue for it. If we just want to refactor it we should open refactor issue. I think that this PR is not a right place to fix this, because we didn't introduced this naming in this PR

Reply:
@NejcZdovc that seems fair but pls open an issue to track it.

Reply:
issue created https://github.com/brave/browser-laptop/issues/10989